pydoctor/themes/base/search.js

```diff
@@ -1,5 +1,14 @@
 // This file contains the code that drives the search system.
 // It's included in every HTML file.
+// Ideas for improvments: 
+// - Include filtering buttons:
+//    - search only in the current module
+//    - have a query frontend that helps build complex queries
+// - Filter out results that have score > 0.001 by default and show them on demand.
+// - Should we change the default term presence to be MUST and not SHOULD ?
+//        -> Hack something like 'name index -value' -> '+name +index -value'
+//        ->      'name ?index -value' -> '+name index -value'
+// - Highlight can use https://github.com/bep/docuapi/blob/5bfdc7d366ef2de58dc4e52106ad474d06410907/assets/js/helpers/highlight.js#L1
```

Comment:
Indexing docstrings for projects like Twsited might not be best idea becaue of the overhead it creates at searching time. 

```suggestion
// - Highlight can use https://github.com/bep/docuapi/blob/5bfdc7d366ef2de58dc4e52106ad474d06410907/assets/js/helpers/highlight.js#L1
// - Since searching in docstrings can be long, i.e. a search in Twisted docs currently takes few seconds.
//    It would be good to be able to search in an index with names only first, and launch docstring search optionnaly.
```